tests/cupy_tests/sorting_tests/test_search.py

```diff
@@ -743,3 +758,94 @@ def test_nonint_sorter(self):
             sorter = xp.array([], dtype=xp.float64)
             with pytest.raises(TypeError):
                 xp.searchsorted(bins, x, sorter=sorter)
+
+
+@testing.gpu
```

Comment:
This annotation is there for historical reasons, no longer needed for new tests :)
```suggestion
```